### DISCUSSION TITLE
Are you still a SageTV user?

### DISCUSSION THREAD

**Discussion by valnar1** [OpenSageTV/Community#18] [Polls]
I realize this question is self-fulfilling since you are unlikely to be here if you weren't, but nevertheless, add your +1 if you are!

I'd like to get an idea of how many of us are out there.

**Comment by Sagetvmaster** [comment]
20 years running?

**Reply by PhilipCRoberts** [reply]
Date of Birth: September 7, 1957 (65)
Join Date: 09-29-2004
Total Posts: 452

**Comment by jusjoken** [comment]
82 unique visitors as of today...May 20th 2023

**Comment by nyplayer** [comment]
Barely use it now ... just have it running but really no longer use for everyday recording mostly because SageTV organizing of movies and series is lacking...All recordings go in 1 directory.

**Comment by t4runnr** [comment]
As I've stated in the old forum site, I use SageTV as a media player only.  I don't have tuners or record directly from SageTV.  I could use plex, kodi, or others but I like how I can customize the organizing of my media on Sage more than any of the other software.

**Comment by Jeff-Haas** [comment]
Still using it as my PVR.  I moved to Sage from BeyondTV years ago!  It's running on Win7 in an old Antec HTPC case.

**Reply by cat6man** [reply]
i also went from beyondTV to sageTV, and still using it daily, running on a linux NUC

**Reply by sircupp** [reply]
I also came over from BeyondTV.  Used ATI all in wonder prior to that.  Still using it everyday with cable card and win 7.

**Comment by dlj9999** [comment]
I'm still a heavy SageTV user.  Running on Win11 right now.  I use Plex to watch my shows off my NAS setup.

**Comment by PhilipCRoberts** [comment]
I still use it.  I record and watch daily. I watch it about as much as I stream.

Date of Birth: September 7, 1957 (65)
Join Date: 09-29-2004
Total Posts: 452

**Reply by tiki-2** [reply]
I still use it most days, but have been using my Roku more and more lately.

**Comment by davidk21770** [comment]
Still a heavy user.  Since 2006.
Thank you everyone for keeping a great program!!!

**Comment by pjwerdna** [comment]
Been using SageTV since 2006 and still use it every day. Still running 7.1.9 as well because I'd need all new hardware to run anything later

**Comment by LoisGNS** [comment]
Every single day - both for watching and for recording (multiple tuners). Server is currently on Windows 8.1.  I mostly watch via Placeshifter on my computer in another room, but also use extenders in various parts of the house. Much preferable than streaming, since I get to keep shows locally for as long as I like (use Playon and occasionally Audials to save streaming stuff).

**Comment by Zogg44** [comment]
I still have one instance running for just such an emergency. We use Channels DVR that works MUCH better with captioning and recording from TVE, and Plex for watching other media. SageTV lets me go backwards in the guide though and has a better search capability for looking for future episodes, so I use it for manual recordings sometimes.

**Comment by cfredericksen** [comment]
Easily the best PVR for speed, reliability and user experience. Ive tried them all and I'm still here.

**Comment by mindcue** [comment]
Been using it non stop since the early SD days.  Getting a bit more challenging to keep running with new DirecTV box changes and all, but can't beat it for functionality and flexibility to this day!

**Comment by sic0048** [comment]
I still use it to record and watch cable via a cable card tuner.  Although honestly I am watching broadcast TV less and less in general.  If/when our cable company drops support for cable cards, we might be inclined to drop cable completely.  Once that happens, I'll likely stop using SageTV.

**Comment by valnar1** [comment]
Well, since I started this thread, my own TV experience has changed.  My Cable company _Breezeline_ (formerly _WOW_) decided do away COMPLETELY with their traditional coax based QAM TV service.  All my tuners would die by the end of this month, in favor of their new streaming service.  So I cancelled my TV with them.  I still have an antenna for the locals, but it does hit that nail further into the coffin.

I have a pair of Hauppauge HD PVR tuners to sell now....

**Comment by Ugadata** [comment]
There are some alternatives ways of recording "cable channels" without cable.  You may not be interested but it is possible.

YouTube, Hulu, DirectTV, have a "cable" replacement stream (my terminology).  Sage is not able to connect directly but using OpenDCT with Channels DVR it is possible to still record most "cable" channels.  There are other providers as well PlutoTV, FriendlyTV?, etc they are less expensive but also less content.  The big downside for some is that you can only get stereo audio, no surround sound (5.1 or better)

**Reply by valnar1** [reply]
I'm not sure how Channels DVR would help.  History channel for instance requires a subscription from a legit cable provider.  Granted there are other ways to download such content after it airs, but if you are referring to actual Live TV, I fail to see how it would help.

**Reply by Ugadata** [reply]
Channels DVR works either with an existing Cable subscription or with one of the "cable" replacement subscriptions (Hullu Live, YouTube Live, etc).

Yes, for Channels to work you need a subscription to "something" for content. If that subscription contains "The History Channel" then you can record it with Channels DVR and/or in SageTV through OpenDCT to connect to the Channels DVR server. Channels will be signing into the streaming content using your Content Provider's credentials (similar to what PlayON does). For the Guide Data in Sage, Schedule Direct does support several of the alternate choices or you can use the XMLTV plugin and import schedule data.

For a while I struggled with why I would want to use Channels DVR since I would still need my cable subscription (or something) to provide content. Not cutting and not saving any money, in fact it would slightly increase since Channels is ~$80/yr. By using the alternative cable solution(s) you can save a few dollars (mostly by not having to pay all the various service fees the cable companies are passing on to us.). Although some of the alternatives may not have as much content they may have what you are looking for (PlutoTV, as an example)

**Reply by LoisGNS** [reply]
If I wanted to get started with Channels DVR and OpenDCT (and anything else I might need to make these work), where can I find appropriate tutorials? When I've looked at these casually, one or both seem to depend on things that I'm not familiar with, such as a docker, unraid, and probably a few other things that are obscure to me. I'm on Windows 10 and don't do Java or Python. I do Visual Basic 6 and VBA, for all the good that does in this context, except that I am not completely unaware of coding in general. I briefly played with installing some Python stuff with Git, pip, or things like that, but found it very painful and I didn't really know what I was doing.

**Reply by Ugadata** [reply]
F.W.I.W.  - I don't use unraid or dockers, either.  I am running SageTV ( 9 something), OpenDCT, and Channels DVR under Windows 7 Pro on an Intel NUC (don't remember model).  There is some manual configuring of OpenDCT so it will connect to Channels DVR.  If you have ever modified Sage's property file, not all that different.

Yeah, like NYPlayer alludes to, finding the instructions is a challenge in itself.  It kind of requires a lot of  perusing through some very voluminous threads (in the old website now) just to find the tidbits you want (even those typically span several posts).

Although setting up OpenDCT can be challenging, it isn't all that difficult to do (at least in hindsight).  Although OpenDCT was most likely created to connect with Digital Cable Tuners (Ceton, Silicon Dust, etc.) , there was some forethought to allow for other types of connections (unlike SageDCT) as well.

The OpenDCT thread contains all the needed info needed to setup OpenDCT and the needed command lines needed to connect to Channels DVR.

**Reply by LoisGNS** [reply]
I assume you're referring to the OpenDCT thread on the old forum? I'll look for it. I just looked again at ChannelsDVR and need to study it further to understand what it could do for me. I gather that OpenDCT provides for some integration between Sage and Channels? I'm not looking for a Sage replacement (as long as it keeps working). 

Edit:  I'm peeking at the OpenDCT thread and am not at all clear on what it does. The first post just announces it and describes how to set it up, with various links, but what does it actually do? It seems mostly related to HDHR, but Sage already supports that, so what does OpenDCT add? I have an ancient HDHR, but its tuner is much weaker than my other OTA tuners, so it's way down in  the merits list.

**Reply by Ugadata** [reply]
I am not doing very good job of explaining this, but you are starting to get there.  I gather you have only been using OTA tuners which Sage can see natively and would not need OpenDCT.  The instructions for using OpenDCT for connecting to Channels DVR are pretty far into the OpenDCT thread.

OpenDCT (and SageDCT before it) was mostly a way to use using cable card tuners, which Sage cannot use natively. (HDHR PRIME, not the OTA tuners)   It just happens that some forethought was put into OpenDCT to allow it to connect to other IP streams as well (an IP Camera as one example).

So as an example, SageTV w/OpenDCT is able see and use cable card tuners (HDHR Prime, Ceton, and maybe the Hauppauge Cable Card Tuner).  SageTV <--> OpenDCT <--> CableCardTuner (HDHR Prime or CETON).

Now the case for SageTV and Channels DVR, think of Channels DVR as just another tuner (a software tuner, if it helps).  At this time, similar to the cable tuners, Sage cannot see or work with Channels DVR natively.  However,  Once OpenDCT is configured to use Channels DVR, Sage can see and use the  stream from Channels DVR.
SageTV <--> OpenDCT <--> Channels DVR Server.

**Reply by nyplayer** [reply]
The ChannelsDVR and Opendct is in a separate thread ....

https://forums.sagetv.com/forums/showthread.php?t=66804

Also the new XMLTV Plugin is here ....

https://forums.sagetv.com/forums/showthread.php?t=66899

**Reply by LoisGNS** [reply]
Thanks. I looked at the XMLTV plugin notes and couldn't really understand it. Is it an alternative to OpenDCT or does it do something different? The announcement says, "The changes below should work with most XMLTV grabbers," but what is an XMLTV grabber and what does it do for me? I wish some of these plugin announcements would include some discussion of actual use cases to give people an idea of how much effort to put into getting them working :)

**Reply by Ugadata** [reply]
XMLTV is not an alternative for OpenDCT.  It is for importing EPG data into SageTV.  

If you want to use XMLTV you can but you may not have to.  I still use Schedules Direct.

**Reply by Sagetvmaster** [reply]
Ditto on Schedules Direct.  Best $35 dollars spent each year related to Sage.

**Reply by nyplayer** [reply]
XMLTV is Needed with OPENDCT to get Pluto and other IPTV services into SageTV. If not for NEW XMLTV this would not be possible. It has nothing to do with Schedules Direct. SMH

**Reply by nyplayer** [reply]
https://i.postimg.cc/PfKV5888/Screenshot-2023-07-15-071640.png

**Reply by LoisGNS** [reply]
So XMLTV (on its own or plus opendct?) would let me record scheduled livestreams from sources like Pluto? I can currently use PlayOn to _download_ on-demand stuff from Pluto, but am not sure if the two catalogs are identical or even overlap. Where does ChannelsDVR fit into this picture?

**Reply by LoisGNS** [reply]
@Ugadata, you mentioned Schedules Direct. Is there a way to get Pluto (etc) lineups via SD? I have that for my OTA and cable lineups.

**Reply by Ugadata** [reply]
It might be time to start a new thread for this.

SD does show a PlutoTV-National option, but I don't think it works for SageTV.  I don't use PlutoTV.  I do use ChannelsDVR TV Everywhere from the SD lineups and it does work.  I did some testing early on using PhiloTV and the Philo lienups seemed to work.

---------------------------

Short of just giving up and sticking with what you have,  You will need to use OpenDCT and Channels DVR.  For PlutoTV you are going to use Channels DVR (setup Pluto TV in Channels DVR),  You will need OpenDCT to get the stream(s) from Channels DVR (for our purposes Channels DVR is just another network tuner).  At least for PlutoTV, XMLTV is needed to guide data from ChannelsDVR into Sage.

You indicate you also have a cable subscription?  If so and presuming your cable provider is supported, You maybe be able to use that in Channels DVR for TVE (TV Everywhere).  I was able to use the SD listings for Channels DVR TVE to get guide data and it worked.  This maybe be a way to get started while you get a handle on the different pieces needed to make all this work.

**Reply by nyplayer** [reply]
I agree if it is working Stick with what you have... I would be glad to help but I really do not have the time or the interest in supporting EOL Software.

**Comment by nyplayer** [comment]
The problem is that Documentation is all over the place there never was a real WIKI for SageTV. OpenDCT and the new XMLTV documentation has to somehow be moved over here. These are vital to expanding SageTV usage.

**Reply by eelpout** [reply]
While a formal wiki would be nice (and i believe there was a partial one at some point), everything one needs to know is scattered within the old forum site amongst all the threads and posts. Problem is since it went static, it's a chore to search and find anything now. 🤔

**Reply by valnar1** [reply]
This is an unpopular opinion among us, but it probably doesn't really matter.  Everyone who is going to use Sage already is, and we aren't getting any new converts.

**Comment by weston1968** [comment]
I have a server with 12 tuners and two 8tb drives.  I use four HD300's.  It has been in constant use since 2011.  I use two antennas, one pointed at NYC and the other Connecticut.  One of my four HD300's has just died and I have 3 more coming from a sell on GitHub.  I use win 7 and am taking no updates since it is a single use machine.

**Comment by FoggySageTV** [comment]
We are still using and enjoying SageTV.  Rec'd my v6 licenses in 2008.
Using Spectrum's cable card and while no new channels are available on the cablecard and some old ones have disappeared, we plan to continue using cable card until Spectrum shuts it down.
I would like to have the four channel version of the HDHomeRun but it appears that may never be released.

**Comment by Sagetvmaster** [comment]
I keep trying to find a better, Windows-based recording medium, and I just haven't, at least where traditional cable boxes and satellite receivers go.  Add to that the usually bullet-proof media extenders, and it's still hard to say 'goodbye' to SageTV.  
Other than our TiVo's, it's the easiest system to use on a daily basis, especially when using it remotely.

**Comment by gtallan** [comment]
Still using here too - mostly as a media player but it's also still recording the odd thing via HDHomeRun (OTA). My two HD300s are mostly sitting idle in favor of the android TV app. Looking back, got my original license key in 2005...

**Comment by cat6man** [comment]
i'm still using sagetv with a prime/cablecard combo for almost all my tv recording.
i also use hdhr record/tuner for recording some channels (like fs1) that i can't record with sagetv.
for downloaded video saved on a network drive, i am using kodi as it has better multi-channel audio options (bypass to avr).

plus, of course, sagetv has the greatest EVER plugin, the SageTV Recording Extender (SRE) which is essential for baseball games.

**Comment by kennethtipton** [comment]
Love the program use every day.  Have used it since early 2000s.  Use Spectrum and a cable card.  Have 2 HD200  and 1 nVidia Shield with Sage for Android (not as good as the hd200 but it works) .  I provide TV to my sister in her house Down the street over a Mikrotik 1Gig wirelies connection,  Works Great.  Interface is by far the best out there.

**Comment by Gary0201** [comment]
Yes, and I have 3 hd300 units and a massive amount of recorded media. Use Sage for OTA recording (2 dual-tuner hd homerun tuners).  I also use PlayOn (separate system) to record streamed media (netflix, hulu, etc.), then I move them into a Sage media folder, then use BMT to import them as recorded tv (so everything shows up in one place in Sage).  I have repeatedly looked for an alternate solution (just in case), and honestly nothing seems to be better.  

It's a shame that Sage seems to be dwindling down to the few hold outs.  If only it was better packaged with documentation aimed toward the new user, there may have been a larger base of users to help keep a pool of developers, but sadly most of the existing base and developers seem to have left (but for/to what is unknown).

The existing documentation and how-too guides are either non-existent or scattered around haphazardly in the archaic forums no longer used (or searchable), which will only serve to drive more folks away and deter anyone new from even considering using it.  Sad.

**Reply by graywolfusmc** [reply]
I'm in the same boat. Several HD300s which I love since they just work. Haven't had time to play with any alternatives . I once looked at OpenDCT but was very confusing and hard to find the documentation.

**Reply by wayner9** [reply]
I am kind of late in replying to this, but it is hard to get new users when there is no real option for clients.  The Android TV client development has died out and you can't buy HD300s.  I am happy with my HD300s, but it would be nice if they could do 4K and work with streaming stuff like Netflix.

I am dabbling a bit with Plex, which I have for a long time, but that may be the way I have to go for recorded material, and then just use the cable co boxes for watching TV, which is mainly sports for me.

**Reply by eelpout** [reply]
what ever happened to Joshua (Android client)? he sort of resurfaced on his Patreon a few months ago and then disappeared again.

I switch to Plex for viewing when I'd like CC that has been recorded (depending on source). the CC/subtitle playback is sort of hit and miss with the Android client; _sometimes_ it actually works on my Fire Stick.

**Comment by Polypro1** [comment]
Since 2005!

**Comment by mgpaulus** [comment]
Looks like my earliest recording was done on my mythtv system in 2006 (Yeah, I still have that around, and I can see the recordings in my sagetv box), and my first sagetv recording was done in 2012, so not as long as some, but I've been doing the home PVR/DVR thing for a while also.  Personally, I find the fact that the HD300 I use just works, and the ability to add "favorites" to auto-capture shows (I have a couple of favorites to capture every movie on Hallmark/Hallmark Movies & Mysteries/Hallmark Drama/GAF/UpTV, and they just show up.)  That is my biggest WAF.

**Comment by weston1968** [comment]
Is there an ATS3 tuner available?   My Hauppauge tuners suck.  On the quad
I have to give each tuner its own set of channels. Each tuner acts
differently, but repeatably.

**Reply by nyplayer** [reply]
SageTV does not support ATSC 3 tuners. The Silicondust Flex 4K does ATSC 3 ... You can use them in SageTV but only to record ATSC1

**Comment by weston1968** [comment]
Any plans?

**Reply by nyplayer** [reply]
Plans ? SageTV is no longer officially supported so I doubt, any new features will be developed. Look into NEXTPVR or Channels DVR.

**Comment by weston1968** [comment]
Too bad.  I saw that there were people working on sage.  In NYC a physical
channel is now ATS3. over time sage will become obsolete. I spend
$300/month on streaming, but watch mostly sage

**Comment by TGsell1** [comment]
My OTA outside NYC (quad tuner) is very high quality 90% of time.  During storms it gets a little pixilated etc but otherwise very nice Sent from my iPhoneOn Jan 20, 2024, at 5:26 PM, weston1968 ***@***.***> wrote:﻿
Too bad.  I saw that there were people working on sage.  In NYC a physical

channel is now ATS3. over time sage will become obsolete. I spend

$300/month on streaming, but watch mostly sage

**Comment by mgpaulus** [comment]
I dumped Xfinity/Comcast about 8 months ago.  Instead I got an HDHR5-4US (HDHR Quattro) which I run through Channels.dvr.  I also have a subscription to frndlytv which gives me my Hallmark, etc.  The frndly tv is connected to channels.dvr using the frndlytv adapter provided by Matt Huisman (https://www.matthuisman.nz/2021/11/frndly-tv-for-channels.html).  So, I have my local OTA stuff, and I have a fair amount of the channels I want.  I suppose if I really wanted to, I might be tempted to try the DirectTV Streaming through Channels.dvr if I need to watch additional stuff, which I don't need to do right now.  For SageTV, I use an HD300, which I find is totally adequate.  

My Sagetv box is Windows 7 box (Yeah, I have a new MoBo/CPU/Memory with a new SSD and Windows 11 on it, just been waiting for a free weekend to do the whole cutover), and I currently have the frndly.tv adapter and channels.DVR running on a 2012 Mac Mini running ubuntu, soon to be replaced by an HP EliteBook G3 SFF with an I7/32GB/1TB running Debian 12.4.  

My kid sort of misses some of the sports channels, but I can always pick up Peacock or Paramount+ or whatever on my AppleTV device, where I also have Disney+, Prime & Netflix.  Disney+ I get as part of my Verizon plan, Prime comes from Prime, and I pay for Netfix, although I'm not sure I'm watching enough on there to justify continuing to pay for that also.

Anyway, these are all ways that I've gone about trying to reduce some of my monthly expenses, since Comcast was going over 300/month, and I found that hard to justify for the 4 or 5 channels that we mostly watch.

Now all I need to for Ting Internet to get their installation work done, and I can drop comcast internet and get unlimited 2GB internet for $90 instead of the current $130 that I pay for 1.2GB unlimited.

**Comment by RJ-Make** [comment]
Don't worry, the way things are going every 'Nextgen' channel will be wrapped in DRM. The 'gift' that keeps on giving...

**Comment by weston1968** [comment]
My server is windows 7. I have 12 tuners and two 8TB drives.  I do not
allow any updates so I get no interruptions.  Win 10 and 11 take random
updates

**Comment by Photon0000** [comment]
I'm on new computer hardware and Win 11 with 6 tuners.  Original white HD Homerun model. Same HD 300s. Working better than ever. SageTV forever.

I needed more switch ports and my old cisco was so old and occasionally needing to be rebooted isn't supported anymore so I got a larger switch..... and returned it because it didn't work with the old white HD homerun tuners and got a different brand that also didn't work with them... and returned it and one other brand and gave up. I discovered that these old HD homerun tuners now have to be on a separate mini switch because every large new switch managed or unmanaged I tried doesn't work with them. But every older 5 to 8 port mini switch I have plus a new 5 port switch bought just for these tuners works fine with them.

**Comment by Sl-a-ck** [comment]
Daily use:
HD200s (1) in Master bed room (1) In living room.
Windows client on Intel NUC in Kitchen.
Windows client on main WFH office workstation.
Occasional use:
Windows clients on various laptops.
Place shifter on various laptops.

**Comment by jdaminc** [comment]
Sage user since the beginning.... Down to OTA with 4 tuners and 60 plus channels between Buffalo NY and Toronto. Still have 2 HD200 boxes in use (just lost a HD100 box) and looking to add a couple Mini PCs. Tried Sage on the new fire stick (still crashes after watching a few episodes of anything)....I'm riding this out till the end with a windows 7 server and 40Tb of hard drive storage (so far....)

**Comment by TGsell1** [comment]
The Fire Stick aka Android mini client takes a fair. Bit of tweaking but now works reliably for me.   Sometimes the HD 100 and 200 issues can be traced to faulty caps … it’s an easy fix if you can solder! Sent from my iPhoneOn Apr 4, 2024, at 12:24 PM, jdaminc ***@***.***> wrote:﻿
Sage user since the beginning.... Down to OTA with 4 tuners and 60 plus channels between Buffalo NY and Toronto. Still have 2 HD200 boxes in use (just lost a HD100 box) and looking to add a couple Mini PCs. Tried Sage on the new fire stick (still crashes after watching a few episodes of anything)....I'm riding this out till the end with a windows 7 server and 40Tb of hard drive storage (so far....)

**Comment by jdaminc** [comment]
the power supply caps are good...I did the replacement years ago.....still have to check the other board though.... tried  fire stick tweaking in the past. haven't really messed with the current ver of fire stick. the previous attempts didn't win me over

**Comment by wayner9** [comment]
I have been lucky with my SageTV hardware, it has all been good.  Hauppauge HD-PVR power supplies were notoriously bad, even well after the capacitor plague issue should have disappeared.  Same with the HDHR - I had power supplies die on that also.
